app/models/task.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: tasks
+#
+#  id          :uuid             not null, primary key
+#  name        :string
+#  description :text
+#  state       :integer
+#  project_id  :uuid
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+
+class Task < ActiveRecord::Base
+  belongs_to :project
+
+  enum state: %i{todo in-progress done}
```

Comment:
Definitely not required, but it might have been nice to assign values to the enumeration like in the project. The main reason is it adds flexibility down the road in case it has to change.